examples/custom_object.py

```diff
@@ -78,6 +78,28 @@ def main():
     print("Resource details:")
     pprint(resource)
 
+    # patch the resource
+    resource = api.patch_namespaced_custom_object(
+        group="stable.example.com",
+        version="v1",
+        name="my-new-cron-object",
+        namespace="default",
+        plural="crontabs",
+        body={'image': 'my-awesome-cron-image:2'},
+        content_type="application/merge-patch+json"
```

Comment:
This parameter is not necessary - select_header_content_type can manage with it.

Reply:
Custom resources support JSON patch, so this parameter can be used to hint JSON patch / JSON merge patch, right?

Reply:
Yes, exactly.

Reply:
What about `application/apply-patch+yaml`? Why aren't we allowing to override from the caller?